dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -27,16 +27,21 @@ public class ObjectCreationNode extends Node {
     // Class body for anonymous classes, otherwise null.
     protected final @Nullable ClassDeclarationNode classbody;
 
+    // Enclosing type receiver
+    protected final Node receiver;
+
     public ObjectCreationNode(
             NewClassTree tree,
             Node constructor,
             List<Node> arguments,
-            @Nullable ClassDeclarationNode classbody) {
+            @Nullable ClassDeclarationNode classbody,
+            Node receiver) {
```

Comment:
Can you put the parameter in a position where it follows the syntax? So here, I would put the new `receiver` node between the existing `constructor` and `arguments` parameters?
Also, add an example to the class documentation.
CI will probably also fail because of all the missing javadoc, so please do add something for all the fields and the constructor.

Reply:
Thanks!